data/anomaly/train.json

```diff
@@ -0,0 +1,70 @@
+{
+    "image_path": {
+        "1": "test\/good\/006.png",
+        "2": "test\/good\/002.png",
+        "3": "test\/good\/005.png",
+        "4": "test\/good\/003.png",
+        "5": "test\/good\/009.png",
+        "6": "test\/good\/000.png",
+        "7": "test\/good\/001.png",
+        "8": "test\/good\/007.png",
+        "9": "test\/good\/004.png",
+        "10": "test\/good\/008.png",
+        "11": "train\/good\/013.png",
+        "12": "train\/good\/006.png",
+        "13": "train\/good\/002.png",
+        "14": "train\/good\/014.png",
+        "15": "train\/good\/019.png",
+        "16": "train\/good\/012.png",
+        "17": "train\/good\/005.png",
+        "18": "train\/good\/018.png",
+        "19": "train\/good\/010.png",
+        "20": "train\/good\/003.png",
+        "21": "train\/good\/009.png",
+        "22": "train\/good\/000.png",
+        "23": "train\/good\/016.png",
+        "24": "train\/good\/001.png",
+        "25": "test\/star\/006.png",
```

Comment:
Not sure I understand the format of these annotation files. Why is there `star` category in the training set, instead of only images from the `train/good` folder?

Reply:
Yeah we don't need it. Since we have `val.json` as well, I can distribute these between `test` and `val`.